config/tasks/main.yml

```diff
@@ -2,24 +2,26 @@
 
 - name: disable and kill unattended upgrades to prevent dpkg locks
```

Comment:
Can you amend to something like: "Disable unattended-upgrades and apt-daily services & timers, and wait for in-flight updates to finish"